hedyorg/hedy
# 🚚 add missing Tailwind dependency in `doit` script

Rebuilding Tailwind files should also happen if `styles.css` is changed; this wasn't happening because the dependency was missing.

**How to test**

Run:

```
doit watch frontend
```

When `doit` prints "Watching 280 files", edit and save `styles.css`. Observe that the Tailwind job is executed again.
